app/js/components/Dashboard.jsx

```diff
@@ -0,0 +1,72 @@
+/** This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+import React from "react";
+import { Link } from "react-router";
+
```

Comment:
You do not want to use the `react-router` module directly as most of it's functionalities is now exposed in the `react-router-dom` module.

For more info, please refer to this stack overflow answer here;

https://stackoverflow.com/questions/42684809/react-router-vs-react-router-dom-when-to-use-one-or-the-other

Reply:
Great post @Efosaok.